tools/cli/include/nearobject/cli/NearObjectCliData.hxx

```diff
@@ -13,6 +13,15 @@
 
 namespace nearobject::cli
 {
+/**
+ * @brief Helper structure to hold parameters for a ranging session.
+ */
+struct UwbRangingParameters
+{
+    uint32_t sessionId;
+    std::vector<uwb::protocol::fira::UwbApplicationConfigurationParameter> appConfigParams;

```

Comment:
nit: for structures that are purely data containers, we prefer to give the properties capitalized names. Also, please use complete words:

```suggestion
    uint32_t SessionId;
    std::vector<uwb::protocol::fira::UwbApplicationConfigurationParameter> ApplicationConfigurationParameters;
```